README.md

```
@@ -9,7 +9,7 @@
     <a href="https://www.npmjs.com/package/@zenstackhq/cli?activeTab=versions">
         <img src="https://img.shields.io/npm/v/%40zenstackhq%2Fcli/latest">
     </a>
-    <img src="https://github.com/zenstackhq/zenstack-v3/actions/workflows/build-test.yml/badge.svg">
+    <img src="https://github.com/zenstackhq/zenstack/actions/workflows/build-test.yml/badge.svg">
```

markdownlint-cli2 (0.20.0)

[warning] Images should have alternate text (alt text)

(MD045, no-alt-text)